nixos/modules/config/networking.nix

```diff
@@ -59,6 +64,14 @@ in
       '';
     };
 
+    networking.resolvconfOptions = lib.mkOption {
+      type = types.lines;
```

Comment:
Would a list of strings not would be a better type for this kind of information?